lib/ingestor-api/runtime/tests/test_registration.py

```diff
@@ -97,6 +97,21 @@ def test_validates_missing_collection(
             len(self.db.fetch_many(status="queued")["items"]) == 0
         ), "data should not be stored in DB"
 
+    def test_validates_no_collection_id(
+        self, client_authenticated, collection_exists, asset_exists
```

Comment:
looks like only `self` is being used

Reply:
good catch - I just need to include `asset_exists` fixture because apparently that validation check happens before collection ID.